packages/calypso-e2e/src/lib/components/notifications-component.ts

```diff
@@ -23,6 +14,8 @@ export class NotificationsComponent {
 	 */
 	constructor( page: Page ) {
 		this.page = page;
+		// There is no accessible locator for this panel.
+		this.anchor = page.locator( 'div[id=wpnc-panel]' );
```

Comment:
This is something I've yet to codify in the E2E documentation but follows the precedent set in WordPress/gutenberg ([example](https://github.com/WordPress/gutenberg/pull/45784#discussion_r1189538014)).

Reply:
I see what you meant. TIL about the full accessibility tree in Chrome DevTools!